gui/resources.xml

```diff
@@ -379,6 +387,7 @@ Vocabulary
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Begin editing the current layer in an external application.</property>
           <signal name="activate" handler="begin_external_layer_edit_cb"/>
         </object>
+        <accelerator key="l" modifiers="GDK_CONTROL_MASK"/>
```

Comment:
If we were to move the scrap navigation shortcuts, couldn't we use one of those function keys? The reason I'm saying is that none of the other graphics like Gimp or Krita has this feature. So this is a key difference between the rest of the pack which should be highlighted and given a default shortcut.